src/register-org/components/payment-by-account/payment-by-account.component.spec.ts

```diff
@@ -18,7 +18,11 @@ describe('PaymentByAccountComponent', () => {
     services: [],
     hasPBA: null,
     contactDetails: null,
-    hasRegisteredWithRegulator: null
+    hasRegisteredWithRegulator: null,
+    address: null,
+    organisationType: null,
+    regulatoryOrgType: null,
+    reulatorRegisteredWith: null
```

Comment:
@DavidJayakumar - looks like a typo here; perhaps you meant `regulatorRegisteredWith`?